packages/ca_certificates.rb

```diff
@@ -1,28 +1,40 @@
 require 'package'
 
 class Ca_certificates < Package
-  description 'CA Certificates'
+  description 'Common CA Certificates PEM files'
   homepage 'https://salsa.debian.org/debian/ca-certificates'
   @_ver = '20210119'
-  version @_ver
+  version "#{@_ver}-1"
   compatibility 'all'
   source_url "https://salsa.debian.org/debian/ca-certificates/-/archive/debian/#{@_ver}/ca-certificates-debian-#{@_ver}.tar.bz2"
   source_sha256 'af30b4d9a2c58e42134067d29f0ba6120e5960fd140393d5574d4bdcf5b824d6'
 
   binary_url ({
-     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ca_certificates-20210119-chromeos-armv7l.tar.xz',
-      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ca_certificates-20210119-chromeos-armv7l.tar.xz',
-        i686: 'https://dl.bintray.com/chromebrew/chromebrew/ca_certificates-20210119-chromeos-i686.tar.xz',
-      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ca_certificates-20210119-chromeos-x86_64.tar.xz',
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ca_certificates-20210119-1-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ca_certificates-20210119-1-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/ca_certificates-20210119-1-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ca_certificates-20210119-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-     aarch64: '75d332ca6053e5e12d34eb5d72da6daab50d650f6733d1bd9668bed9af802e30',
-      armv7l: '75d332ca6053e5e12d34eb5d72da6daab50d650f6733d1bd9668bed9af802e30',
-        i686: 'e61ae07f85f64f0c53fa922714ac30579c77db5f2e9fa266e4e6059b502f5475',
-      x86_64: '7a782fdba9f89b077004dab99c15367bb999ed1b19485e32a22be2e79c4f3ac3',
+     aarch64: '412f0ca92aaafb7e45543635fdfcda9bf937a8b659d0a10403f4f7e827ddd98f',
+      armv7l: '412f0ca92aaafb7e45543635fdfcda9bf937a8b659d0a10403f4f7e827ddd98f',
+        i686: '527721877820f2be4f83e13b3aa3b237e1bf1f3263e636241f122d9c001f9c08',
+      x86_64: 'f2fa591a9ec3aa7c43a1fd58081f1f6a1b9ef9f8ee4ae0c520b17202648fd395',
   })
 
+  depends_on 'bash' => :build
```

Comment:
Why is a newer version of bash needed?

Reply:
It's not at all. I'm just trying to avoid as much as possible dependencies on programs that use libraries in /usr/lib64.

At some point if we switch to a new glibc that would be useful...

Reply:
Happy to remove if it is an issue.

Reply:
I'm thinking it may be a problem if Chrome OS is using a different version out-of-the-box.  If there is no specific reason for using it, then I think it should be removed.  Would this require new binaries?  I think maybe?

Reply:
Make sure you remove the bash package before you create new binaries. :)

Reply:
Why new binaries? We're just changing the requirement for building here.... let me just adjust the script used too...

Reply:
ACTUALLY, I'm not even using bash any more... I was previously using a shell script I was running from bash... but that's no longer the case, and the heredoc used now dispenses with that logic entirely, so the bash dep was actually irrelevant and extraneous.

Sigh.